packages/property-grid/src/components/ElementList.tsx

```diff
@@ -170,6 +170,8 @@ export const ElementList = ({
     [onSelect]
   );
 
+  const title = `${PropertyGridManager.translate("element-list.title")} (${instanceKeys.length})`;
```

Comment:
Just want to confirm this change here.
Wouldn't this be different from what data-viz and version-compare display? (the result count at the bottom)